docs/buildah-bud.md

```diff
@@ -657,6 +660,27 @@ will convert /foo into a `shared` mount point.  The propagation properties of th
 mount can be changed directly. For instance if `/` is the source mount for
 `/foo`, then use `mount --make-shared /` to convert `/` into a `shared` mount.
 
+## BUILD TIME VARIABLES
+
+The ENV instruction in a Containerfile can be used to define variable values.  When the image
+is built, the values will persist in the container image.  At times it is more convenient to
+change the values in the Containerfile via a command-line option rather than changing the
+values within the Containerfile itself.
+
+The following variables can be used in conjunction with the `--build-arg` option to override the
+corresponding values set in the Containerfile using the `ENV` instruction.
+
+  * HTTP_PROXY
+  * HTTPS_PROXY
+  * FTP_PROXY
+  * NO_PROXY
+
+In addition, the FROM instruction can also be overridden.  Please note that if there are multiple
```

Comment:
Does this make sense as the default.  I would have thought the more likely request would have been to only change the first?  What happens when they FROMs inside of the Containerfile refer back to previous steps?

Reply:
I'll make it change just the first instance then rather than all.  Any thought on the CLI flag?  `-j-first-from` perhaps?

Reply:
I would just change the first FROM for  now, and ignore the rest, no new flag.  I think that was what was asked for.  Wait until someone else asks for changing all of the FROMs.  At that point you probably need to rewrite your containerfile.

Reply:
The option for imagebuilder is `-from`, and it's specified using `from:` [in OpenShift](https://docs.openshift.com/container-platform/4.6/builds/build-strategies.html#builds-strategy-docker-from-image_build-strategies), so `--from` seems fine.

Reply:
I've changed it to a `--from`  option per @nalind's last comment.